### ISSUE TITLE
Dose anyone meet the numda problem? there is a lot of warning when i running the pointpillar code.  

### ISSUE THREAD

**Issue by WAN96** [traveller59/second.pytorch#272]
Compilation is falling back to object mode WITH looplifting enabled because Function "points_in_convex_polygon_jit" failed type inference due to: Invalid use of Function(<built-in function getitem>) with argument(s) of type(s): (array(float32, 3d, C), (slice<a:b>, list(int64), slice<a:b>))
 * parameterized
In definition 0:
    All templates rejected with literals.
In definition 1:
    All templates rejected without literals.
In definition 2:
    All templates rejected with literals.
In definition 3:
    All templates rejected without literals.
In definition 4:
    All templates rejected with literals.
In definition 5:
    All templates rejected without literals.
In definition 6:
    All templates rejected with literals.
In definition 7:
    All templates rejected without literals.
In definition 8:
    All templates rejected with literals.
In definition 9:
    All templates rejected without literals.
In definition 10:
    TypeError: unsupported array index type list(int64) in (slice<a:b>, list(int64), slice<a:b>)
    raised from /home/wx/anaconda3/envs/pointpillars/lib/python3.7/site-packages/numba/typing/arraydecl.py:71
In definition 11:
    TypeError: unsupported array index type list(int64) in (slice<a:b>, list(int64), slice<a:b>)
    raised from /home/wx/anaconda3/envs/pointpillars/lib/python3.7/site-packages/numba/typing/arraydecl.py:71
This error is usually caused by passing an argument of a type that is unsupported by the named function.
[1] During: typing of intrinsic-call at /home/wx/Downloads/second.pytorch-master/second/core/geometry.py (153)

File "../../second.pytorch-master/second/core/geometry.py", line 153:
def points_in_convex_polygon_jit(points, polygon, clockwise=True):
    <source elided>
        vec1 = polygon - polygon[:, [num_points_of_polygon - 1] +
                                 list(range(num_points_of_polygon - 1)), :]

**Comment by Disfish9527** [comment]
These warnings don't seem to have any impact and can be trained and tested properly. If you're in trouble, you can block these warnings.

**Comment by Disfish9527** [comment]
These warnings don't seem to have any impact and can the network can be trained and tested properly. If you're in trouble, you can block these warnings.

**Comment by Disfish9527** [comment]
These warnings don't seem to have any impact and the network can be trained and tested properly. If you're in trouble, you can block these warnings.

**Comment by WAN96** [comment]
> These warnings don't seem to have any impact and the network can be trained and tested properly. If you're in trouble, you can block these warnings.

Yes,you r right. These warnings don't impact the training. Thank you.

**Comment by micjabbour** [comment]
numba 0.44 started showing some deprecation warnings that mostly help users identify more rooms for optimization, see [here](https://numba.pydata.org/numba-doc/dev/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit) for more details.

You can use numba 0.43.1 if you don't want to see these warnings.
